internal/image/fetcher_test.go

```diff
@@ -192,6 +192,11 @@ func testFetcher(t *testing.T, when spec.G, it spec.S) {
 							h.AssertNil(t, err)
 
 							h.AssertNil(t, img.Save())
+
+							// TODO: delete the following line
```

Comment:
Is this a TODO for the future? Or is it intended to be done as part of this PR?

Reply:
Sorry, it's intended to be part of this PR. 😅
I wish I could change this back to draft mode, I'll still trying to get the tests to pass.